### PR TITLE
Adding new UI to promote button

### PR DESCRIPTION
**Pull request by ernestoSk13**
## Proposed Changes

* New UI proposed for the promote button. 

## Testing Instructions

- [ ] Open calypso and navigate to any blog
- [ ] Click on Tools > Advertising
- [ ] Watch the list and look for the promote button. It should have the same UI as the one shown in this figma ss.

Screenshot 2024-01-23 at 6 39 44 p m

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/my-sites/promote-post-i2/components/post-item/style.scss [hunk 1]
```diff
@@ -113,6 +113,8 @@ body.is-section-promote-post-i2 {
 
 		&__post-promote-button {
 			border-radius: 4px;
+			color: #3858ea;
```
Comment (therocket-gr) [inline client/my-sites/promote-post-i2/components/post-item/style.scss]: i think this is not quite right, we shouldnt hardcode the colors

have you tried to change your color palette and check out the results, (you can do so if you click the bottom right button and account settings) i guess that the colors should be shifting when changing 
Screenshot 2024-01-24 at 18 03 13

like the one on the top right 


Screenshot 2024-01-24 at 18 03 54

Comment (ernestoSk13) [reply client/my-sites/promote-post-i2/components/post-item/style.scss]: Hi @therocket-gr  I saw Figma's file and thought we were gonna use the same color as there... so I should only outline the border and remove the background?

Comment (ernestoSk13) [reply client/my-sites/promote-post-i2/components/post-item/style.scss]: I've fixed this behavior in my last commit @sbarbosa  @therocket-gr

File: client/my-sites/promote-post-i2/components/post-item/style.scss [hunk 2]
```diff
@@ -113,6 +113,9 @@ body.is-section-promote-post-i2 {
 
 		&__post-promote-button {
 			border-radius: 4px;
+			border: 1.5px solid var(--color-accent) !important;
```
Comment (sbarbosa) [inline client/my-sites/promote-post-i2/components/post-item/style.scss]: I think the `! important` here is not needed. It is always better to try and avoid using `important` if possible.
Same for the one below, I tested it in the browser and it appears to be working fine

Screenshot 2024-01-24 at 16 39 43

**Review by sbarbosa — APPROVED** [review]
The code looks good @ernestoSk13 👍

I found a minor improvement that we can make, but it is a non-blocking comment.

I also tested this change in Jetpack Blaze and the buttons there continue to look good.

File: client/my-sites/promote-post-i2/components/post-item/style.scss (hunk 2 above)
Comment (ernestoSk13) [reply client/my-sites/promote-post-i2/components/post-item/style.scss]: I'll remove it :)

**Pull request merged**